README.md

```diff
@@ -144,10 +144,16 @@ Build from source:
 Install the client
 -------------------
 
+installing via `go get`:
 ```
 go get -u github.com/confluentinc/confluent-kafka-go/kafka
 ```
 
+installing via `dep`:
+```
+dep ensure -add github.com/confluentinc/confluent-kafka-go/kafka
```

Comment:
Hey @lemoney . Are you sure about this change? I mean people are using all the time the different ways on handle the vendor folder. Also you can use `go get` and then when you started to use the package call `dep ensure` instead.